config/hooks/post-merge

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source "$HOME/.profile"
```

Comment:
We could update `PATH` just for poetry here, but I don't think sourcing `.profile` will cause any problems, and it seems better for avoiding similar problems in the future.
```bash
export PATH="$HOME/.poetry/bin:$PATH"
```